Classes/Repository/RepositoryIssuesViewController.swift

```diff
@@ -49,7 +51,34 @@ class RepositoryIssuesViewController: BaseListViewController<NSString>, BaseList
         }
     }
 
-    // MARK: Overides
+    // MARK: Issue filtering
+
+    func filter(_ items: [ListDiffable], _ query: String) -> [ListDiffable] {
+        guard !query.isEmpty else {  return items }
+
+        let searchText = query.lowercased()
```

Comment:
As per https://github.com/rnystrom/GitHawk/pull/503#issuecomment-335028446

We could split the query by space, and remove any that start with `is:` and add a flags enum (array) then in the filter group we can do something like:

`if flags.contains(.open) && item.closed { return false }`
`if flags.contains(.closed) && !item.closed { return false }`

Spitballing but this'll be so helpful and I think with the ability to filter open/closed it'll be even better given how annoying it is to scroll through 500 notifications even though half of them are closed 🙄 In hindsight not my best moment

Reply:
Yeah, I will finish my code for this tomorrow 👍